controllers/apply/expiries/send-expiry-email.js

```diff
@@ -0,0 +1,181 @@
+        'scotland': 'advicescotland@tnlcommunityfund.org.uk',
+        'northern-ireland': 'enquiries.ni@tnlcommunityfund.org.uk',
+        'wales': 'wales@tnlcommunityfund.org.uk'
+    }[country];
+
+    return countryEmail || 'general.enquiries@tnlcommunityfund.org.uk';
+}
+
+function getPhoneFor(country) {
+    const countryPhone = {
+        'scotland': '0300 123 7110',
+        'northern-ireland': '028 9055 1455',
+        'wales': '0300 123 0735'
+    }[country];
+
+    return countryPhone || '0345 4 10 20 30';
+}
+
+module.exports = function sendExpiryEmail(
+    {
+        formId,
+        emailType,
+        unsubscribeToken,
+        applicationId,
+        applicationData = {},
+        expiresAt,
+        username
+    },
+    mockMailTransport = null
+) {
+    const emailConfig = {
+        'awards-for-all': {
+            name: 'application_expiry_afa',
+            template: path.resolve(
+                __dirname,
+                './views/awards-for-all-expiry-email.njk'
+            )
+        },
+        'standard-enquiry': {
+            name: 'application_expiry_standard',
+            template: path.resolve(
+                __dirname,
+                './views/standard-expiry-email.njk'
+            )
+        }
+    }[formId];
+
+    function getProjectCountry() {
+        if (formId === 'awards-for-all') {
+            return get(applicationData, 'projectCountry');
+        } else if (formId === 'standard-enquiry') {
+            const countries = get(applicationData, 'projectCountries');
+            return countries && countries.length === 1
+                ? countries[0]
+                : 'Multiple';
+        }
+    }
+
+    function getBilingualStatus() {
+        if (formId === 'awards-for-all') {
+            return get(applicationData, 'projectCountry') === 'wales';
+        } else if (formId === 'standard-enquiry') {
+            const countries = get(applicationData, 'projectCountries');
+            return countries && countries.includes('wales');
+        }
+    }
+
+    const projectCountry = getProjectCountry();
+    const isBilingual = getBilingualStatus();
+
+    let subjectLine = getSubjectLineForEmail(emailType);
+    // Combine subject lines for bilingual emails
+    subjectLine = isBilingual
+        ? [subjectLine.en, subjectLine.cy].join(' / ')
+        : subjectLine.en;
+
+    const expiresOn = moment(expiresAt);
+
+    const dateFormat = 'D MMMM, YYYY HH:mm';
+    const expiryDates = {
+        en: expiresOn.format(dateFormat),
+        cy: expiresOn.locale('cy').format(dateFormat)
+    };
+
+    const baseLink = `/apply/emails/unsubscribe?token=${unsubscribeToken}`;
+
+    const templateData = {
+        isBilingual: isBilingual,
+        projectName: get(applicationData, 'projectName'),
+        countryPhoneNumber: getPhoneFor(projectCountry),
+        countryEmail: getEmailFor(projectCountry),
+        unsubscribeLink: {
+            en: `https://tnlcommunityfund.org.uk/${baseLink}`,
```

Comment:
Can we add `www.` in here? You also don't need the trailing slash before `${baseLink}` (it starts with a slash!)